modules/tinymce/src/plugins/autolink/test/ts/browser/ConsecutiveLinkTest.ts

```diff
@@ -1,41 +1,37 @@
-import { Log, Pipeline, Step } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
-import { TinyApis, TinyLoader } from '@ephox/mcagar';
+import { before, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
 import Env from 'tinymce/core/api/Env';
-import AutolinkPlugin from 'tinymce/plugins/autolink/Plugin';
+import Plugin from 'tinymce/plugins/autolink/Plugin';
 import Theme from 'tinymce/themes/silver/Theme';
 import * as KeyUtils from '../module/test/KeyUtils';
 
-UnitTest.asynctest('browser.tinymce.plugins.autolink.ConsecutiveLinkTest', (success, failure) => {
-
-  Theme();
-  AutolinkPlugin();
-
-  TinyLoader.setupLight((editor, onSuccess, onFailure) => {
-    const tinyApis = TinyApis(editor);
-    const steps = Env.browser.isIE() ? [] : [
-      tinyApis.sFocus(),
-      Log.stepsAsStep('TBA', 'AutoLink: Chrome adds a nbsp between link and text', [
-        tinyApis.sSetContent('<p><a href="http://www.domain.com">www.domain.com</a>&nbsp;www.domain.com</p>'),
-        tinyApis.sSetCursor([ 0, 1 ], 15),
-        Step.sync(() => {
-          KeyUtils.type(editor, ' ');
-        }),
-        tinyApis.sAssertContent('<p><a href="http://www.domain.com">www.domain.com</a>&nbsp;<a href="http://www.domain.com">www.domain.com</a>&nbsp;</p>')
-      ]),
-      Log.stepsAsStep('TBA', 'AutoLink: FireFox does not seem to add a nbsp between link and text', [
-        tinyApis.sSetContent('<p><a href="http://www.domain.com">www.domain.com</a> www.domain.com</p>'),
-        tinyApis.sSetCursor([ 0, 1 ], 15),
-        Step.sync(() => {
-          KeyUtils.type(editor, ' ');
-        }),
-        tinyApis.sAssertContent('<p><a href="http://www.domain.com">www.domain.com</a> <a href="http://www.domain.com">www.domain.com</a>&nbsp;</p>')
-      ])
-    ];
+describe('browser.tinymce.plugins.autolink.ConsecutiveLinkTest', () => {
+  before(function () {
```

Comment:
I feel like I've seen this in a utility function before, but I might just be confusing it with the phantom-js skipper. If it is in a utility function, can we use that instead here (and if not, should we put it in one?)?

Reply:
It really doesn't make sense for this to be a utility function, as it varies based on the browser type and you can have various different conditions.